### ISSUE TITLE
PHP Method's with Nullable Return Types Breaks with PSR-2

### ISSUE THREAD

**Issue by Labradoodle-360** [squizlabs/PHP_CodeSniffer#1631]
When a method contains a nullable return type:
```php
public function helloWorld() :? string
```

PHPCS breaks when running with PSR-2. This is similar to nullable scalar types in method arguments which was already addressed (https://github.com/squizlabs/PHP_CodeSniffer/issues/1279). Hopefully this can get addressed shortly.

**Comment by gsherwood** [comment]
Can you provide any more info about how PHPCS breaks for you? If I run it over that code, nothing appears broken.

**Comment by Labradoodle-360** [comment]
I apologize for the false alarm, after digging deeper into the code, something else was causing this error -- PHP syntax check that was not accepting PHP 7.1 syntax. Thanks for the quick response.
